src/textCampaign/services/textCampaign.service.ts

```diff
@@ -0,0 +1,126 @@
+import { Injectable, Logger, NotFoundException } from '@nestjs/common'
+import { PrismaService } from 'src/prisma/prisma.service'
+import { CreateProjectSchema } from '../schemas/createProject.schema'
+import { RumbleUpService } from './rumbleUp.service'
+import { TextCampaignStatus } from '@prisma/client'
+
+@Injectable()
+export class TextCampaignService {
+  public readonly logger = new Logger(TextCampaignService.name)
+
+  constructor(
+    private readonly rumbleUpService: RumbleUpService,
+    private readonly prisma: PrismaService,
```

Comment:
If you have this class use `createPrismaBase` then you don't have to declare a logger and you don't have to inject the `PrismaService`. It's all done for you.

https://github.com/thegoodparty/gp-api/blob/8f4d0a99264f0705aa0c616050b97744d95192c8/src/campaigns/services/campaigns.service.ts#L29